api/src/test/java/com/staxpayments/api/repository/InvoiceRepositoryTest.kt

```diff
@@ -0,0 +1,88 @@
+package com.staxpayments.api.repository
+
+import com.staxpayments.api.datasource.InvoiceFakeRepository
+import com.staxpayments.api.requests.CreateInvoiceBody
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class InvoiceRepositoryTest {
+
+    private lateinit var invoiceId: String
+    private lateinit var fakeRepository: InvoiceFakeRepository
+
+    private val fakeBody = CreateInvoiceBody(
+        total = 2.09,
+        url = "https://app.staxpayments.com/#/bill/",
+        meta = null,
+        files = null,
+        customerId = "ac097e4f-2a99-4689-8f90-29ecc2ff8455",
+        sendNow = true,
+        isPartialPaymentEnabled = true
+    )
+
+    @OptIn(ExperimentalCoroutinesApi::class)
+    @Before
+    fun setUp() {
+        fakeRepository = InvoiceFakeRepository()
+
+        runTest(UnconfinedTestDispatcher()) {
+            invoiceId = fakeRepository.createInvoice(fakeBody).id
+        }
+    }
+
+    @OptIn(ExperimentalCoroutinesApi::class)
+    @Test
+    fun test_createInvoice() = runTest(UnconfinedTestDispatcher()) {
+        val body = CreateInvoiceBody(
+            total = 0.09,
+            url = "https://app.staxpayments.com/#/bill/",
+            meta = null,
+            files = null,
+            customerId = "ac097e4f-2a99-4689-8f90-29ecc2ff8455",
+            sendNow = true,
+            isPartialPaymentEnabled = true
+        )
+
+        val response = fakeRepository.createInvoice(body)
+        invoiceId = response.id
+
+        Assert.assertNotNull(response)
```

Comment:
we can remove static import here` assertNotNull(response)` instead of Assert.assertNotNull(response)

Reply:
Try to update for the rest also